### DISCUSSION TITLE
2024-03-14 12:29:05.997 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as huyixi." 2024-03-14 12:29:31.522 [info] [auth] Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by huyixi** [community/community#112401] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

2024-03-14 12:34:08.960 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-14 12:34:09.007 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as huyixi."

**Comment by ChristLZS** [comment]
me too

**Comment by davecheney** [comment, marked as answer]
@huyixi @ChristLZS please contact support, we cannot assist you via this forum. https://support.github.com/contact.

**Comment by xingbofeng** [comment]
same problem

前面的两位解决了这个问题吗
